src/python/pants/engine/fs.py

```diff
@@ -57,15 +57,15 @@ def with_match_error_behavior(self, glob_match_error_behavior):
       glob_match_error_behavior=glob_match_error_behavior)
 
   @staticmethod
-  def create(relative_to, include, exclude=tuple(), glob_match_error_behavior=None):
+  def create(include, exclude=tuple(), glob_match_error_behavior=None, relative_to=''):
```

Comment:
Now that `relative_to` is not a required argument, I believe that you could just switch this to an override of `__new__` rather than a factory method? ie, remove `PathGlobs.create(..)` in favor of just `PathGlobs(..)`.